Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM node:14-buster-slim as builder
 
 WORKDIR /usr/local/lib
 
-COPY package.json yarn.lock ./
+COPY package.json yarn.lock Gemfile Gemfile.lock ./
```

Comment:
this should not get copied here -- this invalidates every layer when the gemfile changes but it is not used until a much much later layer